wrappers/services.go

```diff
@@ -614,44 +616,77 @@ func (es *ensureSnapServicesContext) ensureSnapSystemdUnits(snapInfo *snap.Info,
 		return nil
 	}
 
+	// lets sort the service list before generating them for
+	// consistency when testing
+	services := snapInfo.Services()
+	sort.Slice(services, func(i, j int) bool {
+		return services[i].Name < services[j].Name
+	})
+
+	var svcQuotaMap map[string]*quota.Group
+	if opts.QuotaGroup != nil {
+		svcQuotaMap = opts.QuotaGroup.ServiceMap()
+	}
+
 	// note that the Preseeding option is not used here at all
-	for _, app := range snapInfo.Apps {
-		if !app.IsService() {
+	for _, svc := range snapInfo.Services() {
+		// if an inclusion list is provided, then we want to make sure this service
+		// is included.
+		// TODO: add an AppInfo.FullName member
+		fullServiceName := fmt.Sprintf("%s.%s", snapInfo.InstanceName(), svc.Name)
```

Comment:
(nitpick) I saw something similar before, I wonder if we need something like`func (app *AppInfo) FullServiceName()` ?

Reply:
Agreed! I had added an TODO right above, and I've added a TODO on my queue to add one! It will be in a follow-up